.golangci.yml

```diff
@@ -87,8 +87,6 @@ linters-settings:
         alias: apiextensionsv1
       - pkg: k8s.io/apimachinery/pkg/apis/meta/v1
         alias: metav1
-      - pkg: k8s.io/apimachinery/pkg/api/errors
-        alias: apierrors
```

Comment:
Still used for imports, so why not lint it anymore?

Reply:
This was a duplicate import in the file, so removed, otherwise we will get below error
```
ERRO [linters_context] invalid configuration, multiple aliases for the same package: pkg=k8s.io/apimachinery/pkg/api/errors aliases=[apierrors,apierrors] 
ERRO [linters_context] invalid configuration, multiple packages with the same alias: alias=apierrors packages=[k8s.io/apimachinery/pkg/api/errors,k8s.io/apimachinery/pkg/api/errors] 
```